relay-config/src/config.rs

```diff
@@ -441,10 +441,13 @@ pub struct Relay {
     /// The port to bind for the unencrypted relay HTTP server.
     pub port: u16,
     /// Optional port to bind for the encrypted relay HTTPS server.
+    #[serde(skip_serializing)]
     pub tls_port: Option<u16>,
     /// The path to the identity (DER-encoded PKCS12) to use for TLS.
+    #[serde(skip_serializing)]
     pub tls_identity_path: Option<PathBuf>,
     /// Password for the PKCS12 archive.
+    #[serde(skip_serializing)]
     pub tls_identity_password: Option<String>,
```

Comment:
If we don't support them, can we remove these options?

Reply:
So, the one reason I still kept this, because of the old Relays  - in-case they will get updated and we want to ensure the message is delivered that TLS is not supported anymore, see https://github.com/getsentry/relay/blob/4eecaba5db4d076bfc66552fdde0fd2aff27f3c7/relay-server/src/actors/server.rs#L50-L55

But I'm also fine if we ok with removing all this altogether. I can include the cleanup in my PR as well.

Reply:
If we keep the config to indicate they are no longer supported, I'd log a message to indicate it to users. If not, I'd remove the options too.

Reply:
@olksdr I was missing that piece of context, I'm fine with keeping the options then!